prog/rand.go

```diff
@@ -138,15 +138,37 @@ func (r *randGen) randPageCount() (n uint64) {
 }
 
 // Change a flag value or generate a new one.
-func (r *randGen) flags(vv []uint64, bitmask bool, oldVal uint64) (v uint64) {
+func (r *randGen) flags(vv []uint64, size uint64, bitmask bool, oldVal uint64) (v uint64) {
```

Comment:
We now don't use `size`, please revert this part of changes.

Reply:
Done.